src/schema/objects/metadata.yaml

```diff
@@ -2429,6 +2589,13 @@ PulseSequenceType:
     (for example, `"MPRAGE"`, `"Gradient Echo EPI"`, `"Spin Echo EPI"`,
     `"Multiband gradient echo EPI"`).
   type: string
+PupilFitMethod:
+  name: PupilFitMethod
+  display_name: Pupil Fit Method
+  description: |
+    The method employed for fitting the pupil:
+    `"centre-of-mass"` or `"ellipse"`.
```

Comment:
If you think the list is pretty comprehensive, then I would recommend using an enum. Otherwise, maybe you could make it more clear that these two options are just examples? 

Something like the following, maybe?

```suggestion
    for example, `"centre-of-mass"` or `"ellipse"`.
```

Reply:
I will go with the example option and mention the recommendation to use these exact labels, as you mention below. Thanks.